### ISSUE TITLE
Remove/modify misleading `togglePaneZoom` documentation

### ISSUE THREAD

**Issue by jjspace** [microsoft/terminal#7812]
I recently started playing with Panes in Terminal and immediately wanted the Pane Focus Zoom feature and went looking for it. I see it was added but then turned off in stable due to problems leaving a blank screen as noted in #7252 and #7630. The documentation directly on the panes page itself (https://docs.microsoft.com/en-us/windows/terminal/panes) doesn't talk about it which is good but the documentation on the Custom Actions page (https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#zoom-a-pane) still show it as an option. I tried using this getting very confused why it wasn't working until I found the PR previously mentioned.

**TL;DR** `togglePaneZoom` is currently disabled in Stable but documentation [here](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#zoom-a-pane) seems to say otherwise.

**Requested Fix:** Remove documentation on this page OR (personally preferred) Add a note next to the command listing saying it's only currently available in Preview builds

**Comment by DHowett** [comment]
Whoops! Thanks. I moved this over to https://github.com/MicrosoftDocs/terminal/issues/150
